docker_cli/_docker_mamba.py

```diff
@@ -0,0 +1,235 @@
+from __future__ import annotations
+
+import os
+import shlex
+import textwrap
+from typing import Iterable, Optional, Tuple, overload
+
+
+def mamba_install_dockerfile(
+    env_specfile: os.PathLike[str] | str = "spec-file.txt",
+) -> Tuple[str, str]:
+    """
+    Creates and returns a Dockerfile for installing micromamba.
+
+    Parameters
+    ----------
+    env_specfile : os.PathLike, optional
+        The path to the specfile, by default "spec-file.txt"
```

Comment:
Let's rename this to "requirements file" per our [earlier discussion](https://github.com/isce3-testing/image-sandbox/pull/2#discussion_r1229921848).

```suggestion
        The path to the requirements file, by default "spec-file.txt"
```